lib/game.rb

```diff
@@ -0,0 +1,49 @@
+require_relative '../lib/game_state'
+require_relative '../lib/game_presenter'
+require_relative '../lib/messages'
+require_relative '../lib/board'
+require_relative '../lib/validator'
+require_relative '../lib/game_settings'
+
+class Game
+    include Validator
+    include Messages
+
+    attr_accessor :game_state, :game_presenter
+
+    def initialize(game_presenter: ConsoleIO.new())
+        @game_presenter = game_presenter
+    end
+
+    def set_game_settings(settings:)
```

Comment:
How else can we *inject* your game settings without this setter method?

Reply:
Do you mean .... moving all this logic into a SetGameSettings class, that calls the different Game Settings? then using the results in the `start()` ?